src/Standards/Generic/Tests/CodeAnalysis/UselessOverridingMethodUnitTest.1.inc

```diff
@@ -64,6 +64,17 @@ class Bar {
     public function sameNumberDifferentParameters($a, $b) {
         return parent::sameNumberDifferentParameters($this->prop[$a], $this->{$b});
     }
+
+    public function differentCase() {
+        return parent::DIFFERENTcase();
+    }
+
+    public function nestedFunctionShouldBailEarly() {
+        function nestedFunctionShouldBailEarly() {
+            // Invalid code needed to ensure an error is NOT triggered and the sniff bails early when handling nested function.
```

Comment:
☝🏻 These comments aren't really needed. (though don't hurt either, so no need to remove them, just wanted to point it out)

Reply:
I added this comment as, to me, it is not immediately obvious why calling `parent::nestedFunctionShouldBailEarly()` is needed inside the function, and in the future, others might wonder the same as this is a fatal error.

Reply:
Well, the test wouldn't be a valid test without the `parent::...()` call as it wouldn't fail if the code causing the sniff to bail early would be removed....

Either way, I'm fine with leaving the comments be